### DISCUSSION TITLE
Option to have the filling of the template guided or direct.

### DISCUSSION THREAD

**Discussion by NuttCorp** [docusealco/docuseal#31] [Ideas]
By default the filling of the template by the user is guided by the app and goes field to field in order. In a situation where there is a lot of check boxes it would be easiest for the user the directly interact with the template. This can be done by minimizing the dialog but would be nice were this can be set as a default behavior on a template level. A related point the dark purple object title gets in the way of other check boxes when they are small and tightly packed.
![DocuSeal__Open_Source_Document_Filling_and_Signing-20230727-222736](https://github.com/docusealco/docuseal/assets/34084918/bfc1b62b-d0dc-4059-a015-c5b3a1d9725d)

**Comment by AlexBTurchyn** [comment]
@NuttCorp Oh, you are right, I didn't think that there could be so many checkboxes. I will try to find a solution and when it is ready, I will let you know
